pkg/sabactl/cmd/root.go

```diff
@@ -13,9 +14,11 @@ import (
 )
 
 var (
-	flagServer string
-
-	api *client.Client
+	flagServer    string
+	flagTLSServer string
+	flagInsecure  bool
+	api           *client.Client
+	tlsApi        *client.Client
```

Comment:
In the `docs/api.md`, you defined the `HTTP API` and `HTTPS API`.
We need to use that terminology as much as possible in  source code.
```suggestion
	httpApi           *client.Client
	httpsApi        *client.Client
```